### DISCUSSION TITLE
Some queries

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#42] [General]
How does one create 
(1) a crescendo pedal?
(2) a crescendo cancel piston?
(3) a reed cancel piston?  ---------> I created a general piston with reed selection, each set to default off and it worked!
(4) a piston to activate a row of pedal pistons?
(5) and of course the additional row of pedal stops (that work independently)?

**Comment by larspalo** [comment]
1. Create a setter element of type Swell. There are four crescendos possible to create A, B, C and D. Each has a maximum of 32 steps. No possibility to control them directly from odf. But you can create (export) a settings file with them pre programmed that has the same name as the odf and place it in the same directory as the odf and then it will be automatically detected and used as default by GO.
2. Not really possible in an easy way that I know of. Just pressing any other combination would "cancel" current crescendo swell. There's no way to easily "disable" and "enable" a crescendo swell in GO.
3. Your way should work at least partly - to have it stay functionally active even when other combinations are activated (which is likely how the real organ behaves) is another thing. Normally it would have been a job for switches, but in your particular case it's somewhat more complicated as the stops are not operated by AND function.
4. A button to activate? You mean a button that allows them to be used (pushed) at all? Sorry, not sure how to do that. Possibly again a switch network with transparent combinations...
5. Just add another activating and selecting layer of switches that only will be connected to (referenced by) the pedal stops.

For GOODF I'm working on a way to re-order, or move, switches with drag and drop. That is to be able to change their order. A switch can (currently in GO) only reference a switch with a lower number which makes the order of appearance important, and a real pain when needing to expand or alter a switch network.
